lib/rubocop/cop/rspec/context_method.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # `context` should not be used for specifying methods.
+      #
+      # @example
+      #   # bad
+      #   context '#foo_bar' do
+      #     # ...
+      #   end
+      #
+      #   context '.foo_bar' do
+      #     # ...
+      #   end
+      #
+      #   # good
+      #   describe '#foo_bar' do
+      #     # ...
+      #   end
+      #
+      #   describe '.foo_bar' do
+      #     # ...
+      #   end
+      class ContextMethod < Cop
+        MSG = 'Use describe for testing methods.'.freeze
+        METHOD_INDICATORS = %w[. #].freeze
+
+        def_node_matcher :context_method, <<-PATTERN
+          (block (send #{RSPEC} { :context } $(str #method_name?) ...) ...)
+        PATTERN
+
+        def on_block(node)
+          context_method(node) do |context|
+            add_offense(context, message: MSG)
+          end
+        end
+
+        def autocorrect(node)
+          lambda do |corrector|
+            corrector.replace(node.parent.loc.selector, 'describe')
+          end
+        end
+
+        private
+
+        def method_name?(description)
+          METHOD_INDICATORS.include?(description[0])
```

Comment:
This is a bit confusing, and not obvious that `description` is a string. My mistake was that I thought it might be some kind of a node.

WDYT of:
```
description.start_with?(*METHOD_INDICATORS)
```